account-kit/react/src/tailwind/components/modal.ts

```diff
@@ -2,22 +2,10 @@ import type { ComponentDef } from "../types";
 
 export const modalComponents: ComponentDef = {
   ".modal": {
-    "@apply rounded-2xl": {},
+    "@apply rounded-t-2xl md:rounded-2xl": {},
```

Comment:
what's `rounded-t-2xl`?

Reply:
will just round the top borders, since it's a card flushed to the bottom. i'll adapt the global var to this in a follow-up